sql/catalyst/src/main/java/org/apache/spark/sql/connector/write/Write.java

```diff
@@ -76,4 +77,14 @@ default StreamingWrite toStreaming() {
   default CustomMetric[] supportedCustomMetrics() {
     return new CustomMetric[]{};
   }
+
+  /**
+   * Returns an array of custom metrics which are collected with values at the driver side only.
+   * Note that these metrics must be included in the supported custom metrics reported by
+   * `supportedCustomMetrics`.
+   */
+  default CustomTaskMetric[] reportDriverMetrics() {
```

Comment:
`supportedCustomMetrics` is a noun, how about we name this function `driverSideMetrics`?

Reply:
oh nvm, this is to be consistent with the read side API.